package.json

```diff
@@ -80,10 +79,15 @@
     "eslint-plugin-import": "^2.18.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-react": "^7.17.0",
+    "http-server": "^0.12.1",
     "jest": "^25.1.0",
+    "js-yaml-loader": "^1.2.2",
```

Comment:
Move this back to dependencies. Rule of thumb is if you use it in webpack, then it might be used in a production run of next start. This is required for the global install.